src/api/Flooq.IntegrationTest/DataFlowTest.cs

```diff
@@ -0,0 +1,151 @@
+using System;
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.Net;
+using System.Net.Http;
+using System.Text;
+using System.Threading.Tasks;
+using Flooq.Api.Models;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using Newtonsoft.Json;
+using NuGet.Protocol;
+using Factory = Flooq.IntegrationTest.FlooqWebApplicationFactory<Program>;
+
+namespace Flooq.IntegrationTest;
+
+[TestClass]
+public class DataFlowTest
+{
+  private HttpClient _client;
+  
+  [TestInitialize]
+  public void Setup()
+  {
+    _client = Factory.Factory.CreateClient();
```

Comment:
`Factory.Factory` Looks a bit opaque. I would call it `FlooqWebApplicationFactory` then it's clear what kind of factory it is and where it comes from.